src/main/java/com/exasol/projectkeeper/PomFileTemplateRunner.java

```diff
@@ -26,7 +26,10 @@
  */
 public class PomFileTemplateRunner {
```

Comment:
The pom parsing logic should be extracted from this class. Probably together with the writing part.